README.md

```diff
@@ -1,59 +1,88 @@
+## Built With
 
-![screenshot]()
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
 
-## Built With
+To get a local copy up and running follow these simple example steps
 
-- HTML and CSS
+### Prerequisites
 
-## Getting Started
+The basic requirements for building the executable are:
 
-**In this project, students at Microverse will get familiar with CSS, HTML, git/github/github flow and linting**
+- A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+- VSCode or any other equivalent code editor
+- Node Package Manager (For running, debuging, or installing dependancies to be used)
 
-To get a local copy up and running follow these simple example steps.
+# Getting Started
 
-### Prerequisites
+#### Cloning the project
+
+```
+git clone  https://github.com/mwafrika/Capstone-project-v2 <Your-Build-Directory>
+```
+
+## Getting packages and debuging with Stylelint
 
-#### Have nodejs an npm installed locally
+```
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
 
-### Setup
+##### For validation detection using Stylelint Run
 
-#### To get the copy of this project, please go ahead and follow the steps below:
+```
+npx stylelint "**/*.{css,scss}"
+```
 
-- Open your terminal
-- clone the project from the source repository by running run: `git clone https://github.com/mwafrika/portfolio-microverse.git`
-- Open the index.html file in the browser or use the live server extension for vscode.
+##### from parent source directory
 
-#### Next step is to open the cloned repository and run the following command to install the project dependencies locally `npm install` in
+## Getting packages and debuging with Webhint
 
-## Authors
+```
+npm init -y
+npm install --save-dev hint@6.x
+```
 
-👤 **Author**
+##### For validation detection using Webhint Run
 
-- GitHub: [@github](https://github.com/mwafrika)
-- Twitter: [@twitter](https://twitter.com/mwafrikamufung1)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mwafrika-mufungizi)
+```
+npx hint .
+```
 
-## Show your support
+## Getting packages and debuging with Eslint
 
-Give a ⭐️ if you like this project!
+```
+npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+```
+
+##### For validation detection using Eslint Run
+
+```
+
+npx eslint .
+
+```
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Thanks to Cindy Shin whose design of cc global summit 2015 was used.
+

```

Comment:
- [ ] You consider the suggestion from the previous reviewer 👍 but you should provide the link to the author (aka this link https://www.behance.net/adagio07) and the original design (aka this link https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Reply:
sure, I added only on the readme. I will also add them to the website

Reply:
When you hit enter, the hamburger disappear. I am going to chek it again to be automatic

Reply:
Dear @mwafrika, you need only add the link to the Readme file.